saleor/graphql/product/filters.py

```diff
@@ -223,10 +223,12 @@ class Meta:
 class AttributeFilter(django_filters.FilterSet):
     # Search by attribute name and slug
     search = django_filters.CharFilter(method=filter_attributes_search)
+    id = GlobalIDMultipleChoiceFilter()
```

Comment:
Can we change it to `ids` since this is a list? We also have `ids` in a few other operations so that would be consistent.

Reply:
I actually can, I thought the filter wasn't supporting the `field_name` parameters. 👍